doc/guides/dts/howtos.rst

```diff
@@ -204,21 +204,32 @@ variable :makevar:`DTC_OVERLAY_FILE` contains a space- or semicolon-separated
 list of overlays. If :makevar:`DTC_OVERLAY_FILE` specifies multiple files, they
 are included in that order by the C preprocessor.
 
-Here are some ways to set it:
-
-1. on the cmake build command line
-   (``-DDTC_OVERLAY_FILE="file1.overlay;file2.overlay"``)
-#. with the CMake ``set()`` command in the application ``CMakeLists.txt``,
-   before including zephyr's :file:`boilerplate.cmake` file
-#. create a ``boards/<BOARD>_<revision>.overlay`` file in the application
-   folder for the current board revision. This requires that the board supports
+Devicetree overlays can be specified by the user directly, in a
+:file:`CMakeLists.txt` file, or applied automatically.
+
+Overlay files are applied according to the following scheme:
```

Comment:
This is verbose without explaining too much. I agree a tech writer review would be better here. @FrancescoSer can you please help?

Reply:
@mbolivar-nordic sure, I'll take a look